common/reviews/api/core.api.md

```diff
@@ -210,6 +211,7 @@ enum ContourType {
 // @public (undocumented)
 type Cornerstone3DConfig = {
     detectGPU: any;

```

Comment:
Wouldn't passing in an instance of `getGPUTier` to `detectGPU` with the `benchmarksURL` option set work?